modules/cluster-logging-rn-5.3.13.adoc

```diff
@@ -0,0 +1,36 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-3-13"]
```

Comment:
I think contrib guide/mod docs requires the _{context} to be added to the id's in modules (even though in the case of Rel Notes, these will probably not be re-used in multiple contexts)

[id="cluster-logging-release-notes-5-3-13_{context}"]

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata